### PR TITLE
Nishidai andaze/issue292

### PR DESCRIPTION
**Pull request by ameio**
To Do
=============================

- [ ] setup proper space 
- [ ] setup proper line between menu 
- [ ] change font size

### CODE REVIEW

**Comment by ameio** [comment]
@nishidai-andaze 
Please wait 
Dont merge this branch

**Comment by ameio** [comment]
@nishidai-andaze 
Please merge this request

**Comment by nishidai-andaze** [comment]
@ameio Thank you, ameio san

Please use draft status if you do not want pull requests to be merged.

**Review by nishidai-andaze — CHANGES_REQUESTED** [review]
@ameio Please refer to https://tailwindcss.com/docs/font-size#customizing-your-theme to set the font size correctly. The name should be much like tailwindcss.

File: themes/andaze/assets/css/tailwind.css [hunk 1]
```diff
@@ -13,14 +13,21 @@ html.is-animating .transition-fade {
...
+  html {
+    font-family: 'Noto Sans JP', "ヒラギノ角ゴ Pro W3", Meiryo, メイリオ, Osaka, "MS PGothic", arial, helvetica, sans-serif;
   }
 
+  .nav-tabs .nav-link.active {
+    @apply border-b-4 border-black text-black font-bold;
+  }
+}
+
+@layer components {
```
Comment (ameio-developer) [inline themes/andaze/assets/css/tailwind.css:25]: Please remove this class

And this below for custome size

text-[11px]

Comment (nishidai-andaze) [reply themes/andaze/assets/css/tailwind.css:25]: @ameio-developer Thank you very much. The correction has been confirmed.

**Review by nishidai-andaze — APPROVED** [review]

**Pull request merged**